bridge/client/app/_components/ktb-uniform-subscriptions/ktb-uniform-subscriptions.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container *ngIf="projectName$ | async as projectName">
+  <ng-container *ngIf="uniformRegistration as uniformRegistration">
+    <button dt-button uitestid="addSubscriptionButton" [routerLink]="['/project', projectName, 'uniform', 'services',  uniformRegistration.id, 'subscriptions', 'add']" class="mb-2" *ngIf="uniformRegistration.canEditSubscriptions(), else disabledButton">
+      <dt-icon name="plus-add"></dt-icon>
+      Add subscription
+    </button>
+    <ng-template #disabledButton>
+      <button dt-button uitestid="addSubscriptionButton" class="mb-2 disabled" [dtOverlay]="updateDistributor">
+        <dt-icon name="plus-add"></dt-icon>
+        Add subscription
+      </button>
+      <ng-template #updateDistributor>
+        Action denied. Please update your distributor to version 0.9.0 or higher
```

Comment:
Rephrase: "Please update your distributor to version 0.9.0 or higher for editing your subscriptions."

Please explicitly use the attribute disabled for the button, even if the button does nothing. I think then the class disabled should also not be needed anymore. If the overlay is not working with the disabled button, consider putting it into a div or span wrapper and putting the overlay there.